File: Models/Admin/AccountIndexRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DMWebLib.Models.Admin
{
    public class AccountIndexRequestModel
    {
        public int UserTypeId { get; set; }
        public int PageNumber { get; set; }
        public int CompanyId { get; set; }
        public string OrderBy { get; set; }
        public string Filter { get; set; }
    }
}

File: Java/src/com/datamotion/Models/Admin/AccountIndexRequestModel.java
package com.datamotion.Models.Admin;

import com.fasterxml.jackson.annotation.JsonProperty;

public class AccountIndexRequestModel {
	@JsonProperty("UserTypeId")
	private int userTypeId;
	@JsonProperty("PageNumber")
	private int pageNumber;
	@JsonProperty("CompanyId")
	private int companyId;
	@JsonProperty("OrderBy")
	private String orderBy;
	@JsonProperty("Filter")
	private String filter;
	
	public int getUserTypeId() { return this.userTypeId; }
	
	public void setUserTypeId(int userTypeId) { this.userTypeId = userTypeId; }
	
	public int getPageNumber() { return this.pageNumber; }
	
	public void setPageNumber(int pageNumber) { this.pageNumber = pageNumber; }
	
	public String getOrderBy() { return this.orderBy; }
	
	public void setOrderBy(String orderBy) { this.orderBy = orderBy; }
	
	public String getFilter() { return this.filter; }
	
	public void setFilter(String filter) { this.filter = filter; }
}
